xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -702,11 +860,21 @@
         <Parameter Name="byteCount" Type="System.UInt32" Index="2" FrameworkAlternate="dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;net-5.0;net-6.0;netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="destination">The destination address to copy to.</param>
-        <param name="source">The source address to copy from.</param>
+        <param name="destination">The managed pointer corresponding to the destination address to copy to.</param>
+        <param name="source">The managed pointer corresponding to the source address to copy from.</param>
         <param name="byteCount">The number of bytes to copy.</param>
         <summary>Copies bytes from the source address to the destination address.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This API corresponds to the `cpblk` opcode.
+
+Both `destination` and `source` are assumed to be aligned to the machine's natural word size. See ECMA-335, Sec. III.3.30 ("cpblk - copy data from memory to memory") for more information.
```

Comment:
Such a hint for alignment is IMO enough. Something like this could be used for the earlier comments about alignment.